hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/exceptions/SchedulerInvalidResourceRequestException.java

```diff
@@ -29,19 +29,19 @@
  */
 @InterfaceAudience.Private
 @InterfaceStability.Unstable
-public class SchedulerInvalidResoureRequestException extends YarnRuntimeException {
+public class SchedulerInvalidResourceRequestException extends YarnRuntimeException {
   private static final long serialVersionUID = 10081123982L;
 
-  public SchedulerInvalidResoureRequestException(String message) {
+  public SchedulerInvalidResourceRequestException(String message) {
     super(message);
   }
 
-  public SchedulerInvalidResoureRequestException(Throwable cause) {
+  public SchedulerInvalidResourceRequestException(Throwable cause) {
     super(cause);
   }
 
-  public SchedulerInvalidResoureRequestException(String message,
-      Throwable cause) {
+  public SchedulerInvalidResourceRequestException(String message,
+                                                  Throwable cause) {
```

Comment:
We need to pay attention to the indentation, usually 5 characters.

Reply:
Thanks for your review. Corrected